package/MDAnalysis/analysis/gnm.py

```diff
@@ -221,6 +222,13 @@ class GNMAnalysis(object):
           `Bonus_groups` is contained in `selection` as this could lead to
           double counting. No checks are applied. Default is ``None``.
 
+    Attributes
+    ----------
+    results : list
+          eigenvalues and eigenvectors
```

Comment:
I'm just having a look at the rendered docs, and it looks a bit .. off? https://mdanalysis--3194.org.readthedocs.build/en/3194/documentation_pages/analysis/gnm.html
(same thing for the other ones).

I'm not sure why though. Maybe you can just try putting a `.. deprecation` and see if that fixes it?

i.e.
```
eigenvalues and eigenvectors
.. deprecation:: 1.1.0
   The structure of the `results` list will change in MDAnalysis 2.0
```

I can't guarantee it'll fix it, but if I remember correctly we had something similar set up for the `save` deprecations prior to 1.0.

Reply:
yes it looks weird. Sorry but somehow I can't build the docs locally...

Reply:
No worries, I'll look into it on my end, see if I can come up with a reasonable way to format this.

Reply:
Try a new line before ..deprecation?

Reply:
yeah that seems to do the trick on my end